### ISSUE TITLE
[model_to_estimator] estimator not evaluate all outputs defined in keras model but only one

### ISSUE THREAD

**Issue by Jasonnor** [tensorflow/tensorflow#32606]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.13.1
- Python version: 3.5.2
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: CUDA 10.0
- GPU model and memory: 

**Describe the current behavior**
When using keras model with **multiple outputs**, and convert it to estimator by `model_to_estimator`, it only evaluate one output by `estimator.evaluate` or `tf.estimator.train_and_evaluate`. 
And don't know which output it evaluate (it doesn't display output name but only metrics name).

**Describe the expected behavior**
It should only evaluate all outputs defined in keras model, not only pick one.

**Code to reproduce the issue**
A example here:
```python
    ...

    model = tf.keras.Model(inputs=input_layer, outputs=[output_layer1, output_layer2])

    model.compile(optimizer=optimizer_builder.build(config.optimizer),
                  loss={
                      'output_layer1': 'sparse_categorical_crossentropy',
                      'output_layer2': 'sparse_categorical_crossentropy'},
                  metrics={
                      'output_layer1': 'accuracy',
                      'output_layer2': 'accuracy'})

    estimator = tf.keras.estimator.model_to_estimator(
        keras_model=model,
        config=run_config,
    )
    train_spec = tf.estimator.TrainSpec(
        input_fn=lambda: train_input_fn(config))
    eval_spec = tf.estimator.EvalSpec(
        input_fn=lambda: eval_input_fn(config))
    tf.estimator.train_and_evaluate(estimator, train_spec, eval_spec)

    # Only evaluate one output and don't know which output it evaluate
```

**Other info / logs**
tf log here, except two outputs log, but only get one:
```
# Training is OK
INFO:tensorflow:global_step/sec: 1.02106
INFO:tensorflow:loss = 0.16959098, step = 33600 (97.938 sec)
INFO:tensorflow:global_step/sec: 0.943188
INFO:tensorflow:loss = 0.17629223, step = 33700 (106.023 sec)
INFO:tensorflow:global_step/sec: 0.848641
INFO:tensorflow:loss = 0.1101246, step = 33800 (117.835 sec)
INFO:tensorflow:global_step/sec: 0.852682
INFO:tensorflow:loss = 0.104427785, step = 33900 (117.277 sec)
INFO:tensorflow:Saving checkpoints for 34000 into /summary/model.ckpt.
...

# Evaluating only evaluate one output
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Evaluation [150/1500]
INFO:tensorflow:Finished evaluation at 2019-09-17-14:14:45
INFO:tensorflow:Saving dict for global step 34000: accuracy = 0.6914724, sparse_categorical_crossentropy = 1.0857606, global_step = 34000, loss = 1.1107497
```

TensorBoard except two outputs scalar, but only get one:
![2019-09-18 11 27 54](https://user-images.githubusercontent.com/7609173/65105491-aaff6000-da07-11e9-916f-473ba7847ba6.png)

**Comment by gadagashwini** [comment]
@Jasonnor, Thanks for reporting the issue.
Will it be possible to provide the standalone code to reproduce the reported issue. Thanks!

**Comment by Jasonnor** [comment]
@gadagashwini Sure! Here is a test code:
https://colab.research.google.com/drive/1qU5uhOfCy1MFtvUA1-8nJdzdgcdbpQvM

I compiled model by:
```python
    model.compile(optimizer=tf.keras.optimizers.SGD(lr=0.0001, momentum=0.9),
                  loss={
                      'output_1': 'sparse_categorical_crossentropy',
                      'output_2': 'sparse_categorical_crossentropy'},
                  metric={
                      'output_1': 'accuracy',
                      'output_2': 'accuracy'})
```
But only get one loss result, not two result of two output. (and don't know why without accuracy metric result?)
```
INFO:tensorflow:Evaluation [100/100]
INFO:tensorflow:Finished evaluation at 2019-09-19-06:39:42
INFO:tensorflow:Saving dict for global step 0: global_step = 0, loss = 12.851205, sparse_categorical_crossentropy = 2.3049202
```

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
